MekHQ/src/mekhq/gui/dialog/CampaignOptionsDialog.java

```diff
@@ -1072,11 +1073,31 @@ public void actionPerformed(java.awt.event.ActionEvent evt) {
         gridBagConstraints.anchor = java.awt.GridBagConstraints.WEST;
         panSubAcquire.add(choiceAcquireSkill, gridBagConstraints);
 
+        DefaultComboBoxModel<String> acquisitionSortPriorityModel = new DefaultComboBoxModel<String>();
+        acquisitionSortPriorityModel.addElement(CampaignOptions.S_BESTFIRST);
+        acquisitionSortPriorityModel.addElement(CampaignOptions.S_WORSTFIRST);
+        acquisitionSortPriorityModel.setSelectedItem(options.getAcquisitionSortPriority());
+        choiceAcquireSortPriority = new JComboBox<String>(acquisitionSortPriorityModel);
+
+        gridBagConstraints = new java.awt.GridBagConstraints();
+        gridBagConstraints.gridx = 0;
+        gridBagConstraints.gridy = 3;
```

Comment:
Let's make this gridy self-incrementing (except for those situations where two items are on the same row of course), rather than with these static numbers.

Reply:
Also, can we space the Acquisition Skill and Buyer Skill Priority dropdowns out a little better? Right now, at least with my "standard windows theme", they're right on top of each other.